src/style.css

```diff
@@ -0,0 +1,132 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 198px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  margin: 32px 19px 40px;
+  height: 134px;
+  width: 160px;
+}
```

Comment:
[STYLES] - Be consistent with your `margins`. Add only top or bottom, don't add both.